polls/templates/polls/details.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
Put all your "base" HTML stuff in a file called `base.html` and "inject" code using blocks. 

This template would then look something like: 

```html
{% extends "base.html" %}

{% block body %}
    <h1>{{ question.question_text }}</h1>
    {% if error_message %}<p><strong>{{% error_message %}}</strong></p>{% endif %}
<form action="{ url 'polls:vote' question.id %}" method="post">
{% csrf_token %}
{% for choice in question.choice_set.all %}
     <input type="radio" name="choice" id="choice {{ forloop.counter }}" value="{{ choice.id }}">
     <label for="choice{{ forloop.counter }}">{{ choice.choice_text  }}</label><br>
{% endfor %}
<input type="submit" value="Vote">
</form>
{% endblock %}